src/BenchmarkDotNet/Diagnosers/EventPipeProfiler.cs

```diff
@@ -17,73 +17,57 @@
 using BenchmarkDotNet.Running;
 using BenchmarkDotNet.Validators;
 using Microsoft.Diagnostics.NETCore.Client;
-using Microsoft.Diagnostics.Symbols;
-using Microsoft.Diagnostics.Tracing;
-using Microsoft.Diagnostics.Tracing.Etlx;
-using Microsoft.Diagnostics.Tracing.Stacks;
-using Microsoft.Diagnostics.Tracing.Stacks.Formats;
 
 namespace BenchmarkDotNet.Diagnosers
 {
     public class EventPipeProfiler : IProfiler
     {
+        public static readonly EventPipeProfiler Default = new EventPipeProfiler();
+
+        private readonly Dictionary<BenchmarkCase, string> benchmarkToTraceFile = new Dictionary<BenchmarkCase, string>();
+        private readonly ImmutableHashSet<EventPipeProvider> eventPipeProviders;
+        private readonly bool performExtraBenchmarksRun;
+
+        private Task collectingTask;
+
         // parameterless constructor required by DiagnosersLoader to support creating this profiler via console line args
-        public EventPipeProfiler() { }
+        // we use performExtraBenchmarksRun = false for better first user experience
+        public EventPipeProfiler() :this(profile: EventPipeProfile.CpuSampling, performExtraBenchmarksRun: false) { }
```

Comment:
Why this constructor has the `performExtraBenchmarksRun` parameter with `false` value as a default. In the second constructor, this parameter has 'true', I think It is not consistent.

Reply:
because this ctor is used when users specify console line arguments and they simply cant control it from command line level.

We do the same thing for ETWProfiler: https://github.com/dotnet/BenchmarkDotNet/blob/e92abf845ac916e449bf9d8ccf1d872204951bec/src/BenchmarkDotNet.Diagnostics.Windows/EtwProfiler.cs#L32

So, in general, we do it to improve first user experience (shorter time to get the results)